src/main/java/Deadline.java

```diff
@@ -1,13 +1,18 @@
 public class Deadline extends Task {
-    protected String by;
+    private String by;
```

Comment:
Perhaps can give the deadline date a clearer name? Right now the name "by" may not be clear to other programmers.